### ISSUE TITLE
Add check box to remove unregistered torrents

### ISSUE THREAD

**Issue by Ghosthree3** [omaralvarez/deluge-autoremoveplus#21]
Would be quite handy as this is the only hole in running this plugin indefinitely right now, torrents that error out because the tracker killed them won't always (sometimes ever) meet the conditions of their removal.

**Comment by omaralvarez** [comment]
Yes, this is an easy one, although the only problem would be that I don't know if trackers have the same types of messages for removed torrents in trackers. Could some one that uses a lot of trackers shine a light on this?

**Comment by Ghosthree3** [comment]
"Error: Unregistered torrent" is extremely common, sometimes there's additional info, but simply filtering for tracker messages with 'unregistered' (insensitive) should catch almost all of it.

**Comment by danielloader** [comment]
Any update on the progress of this issue? Lots of the torrents I end up seeding for 14 days are unregistered and nuked within a day or so of grabbing.

**Comment by Quiksmage** [comment]
I'd really love this as well. It would add efficiency to my setup.

**Comment by danielloader** [comment]
At this point I'd just take a textbox to regex against a tracker status.
